### PR TITLE
Add docs for Connect My Computer

### PR DESCRIPTION
**Pull request by ravicious**
[RFD 133](https://github.com/gravitational/teleport/blob/master/rfd/0133-connect-my-computer.md) describes how Connect My Computer works.

To try Connect My Computer yourself, you can download Connect v14.0.1, then from the three dots menu in the top right pick "Open config file", set `"feature.connectMyComputer": true` and restart the app. Now log in to a v14+ cluster and you should see the icon in the top left.

Connect 14.0.1 includes a bit outdated version of Connect My Computer, but it's mostly just small UI changes and bugfixes. Next week I'll make a fresh tag build and mention that here when it's done.

### CODE REVIEW

File: docs/pages/connect-your-client/teleport-connect.mdx [hunk 1]
```diff
@@ -64,6 +64,147 @@ The top bar of Teleport Connect consists of:
 The **status bar** at the bottom displays **cluster breadcrumbs** in the bottom left, indicating
 which cluster the current tab is bound to, and the **Share Feedback** button in the bottom right.
 
+## Connect My Computer
```
Comment (ravicious) [inline docs/pages/connect-your-client/teleport-connect.mdx:67]: Is this section big enough that it warrants having a separate page, perhaps written as a how-to guide?

I'm not sure about the how-to guide part, as there's not a lot of explaining what to do.

**Review by zmb3 — APPROVED** [review]

File: docs/pages/connect-your-client/includes/connect-my-computer-prerequisites.mdx [hunk 2]
```diff
@@ -0,0 +1,6 @@
+- A macOS or Linux device.
+- Teleport Connect v14.1+, on the same major version or one version behind the proxy version. See
```
Comment (ptgott) [inline docs/pages/connect-your-client/includes/connect-my-computer-prerequisites.mdx:2]:
```suggestion
- Teleport Connect v14.1+, on the same major version or one version behind the Teleport Proxy Service version. See
```

File: docs/pages/connect-your-client/includes/connect-my-computer-prerequisites.mdx [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+  [version compatibility](../../faq.mdx#version-compatibility).
+- A local Teleport user: you authenticate using credentials or passwordless login, you don't
+  use SSO to log in to the cluster.
```
Comment (ptgott) [inline docs/pages/connect-your-client/includes/connect-my-computer-prerequisites.mdx:5]:
```suggestion
- A local Teleport user: you must authenticate using credentials or passwordless login, and not with SSO.
```

File: docs/pages/connect-your-client/teleport-connect.mdx (hunk 1 above)
Comment (ptgott) [reply docs/pages/connect-your-client/teleport-connect.mdx:67]: I think this page is starting to burst at the seams a little bit, but we can leave the reorganization for a separate project!

File: docs/pages/connect-your-client/teleport-connect.mdx [hunk 4]
```diff
@@ -64,6 +64,168 @@ The top bar of Teleport Connect consists of:
...
+that's why it's not listed in the partial. */}
+- Permissions to read and update user objects in the backend (verbs `read` and `update` for [the
+  `user` resource](../access-controls/reference.mdx#teleport-resources)).
+
+The agent runs as the current system user, not as root. Some features are thus not available, such
+as logging in as other system users or [host user creation](../server-access/guides/host-user-creation.mdx).
+
+### Setup and usage
+
+To begin the setup, click on the laptop icon in the top left and select "Connect My Computer". The
```
Comment (ptgott) [inline docs/pages/connect-your-client/teleport-connect.mdx:92]: I think a screenshot would be great after this paragraph.

File: docs/pages/connect-your-client/teleport-connect.mdx [hunk 5]
```diff
@@ -64,6 +64,168 @@ The top bar of Teleport Connect consists of:
...
+access to the node or simply assign the Connect My Computer role to themselves.
+
+We recommend using Connect My Computer only in scenarios where no other user could plausibly gain
+access to the node, such as when exploring a Teleport cluster as its only user or in a home lab.
+</Admonition>
+
+Next, the setup downloads a Teleport agent for your platform and runs `teleport node configure`
+pointed at the current cluster. Once that is done, Connect My Computer starts the agent and waits
+for it to show up in the cluster as an SSH node.
+
```
Comment (ptgott) [inline docs/pages/connect-your-client/teleport-connect.mdx:112]: A screenshot would be good here as well

**Review by ptgott — APPROVED** [review]

File: docs/pages/connect-your-client/teleport-connect.mdx (hunk 5 above)
Comment (ravicious) [reply docs/pages/connect-your-client/teleport-connect.mdx:112]: Added one extra screenshots for a good measure. The first one shows where the Connect My Computer icon is compared to the rest of regular UI, the second one shows the setup and the third one shows the status view.

**Pull request merged**